addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1046,7 +1047,7 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
             const input = inputEls[i];
             if (newValuesText[i] && input.value && !newValuesText.includes(input.value)) {
                 for (const dependentEl of this.formEl.querySelectorAll(
-                        `[data-visibility-condition="${input.value}"][data-visibility-dependency="${inputName}"]`)) {
+                        `[data-visibility-condition="${this._getNameForSelector(input.value)}"][data-visibility-dependency="${this._getNameForSelector(inputName)}"]`)) {
```

Comment:
If I am not mistaken, the method `renderListItems` is never called. Maybe, can we remove it in `master`?

Reply:
Weird indeed, not used in 15.0 anymore (existed up to 14.0). Also one occurence in website_payment.
You can open a pr for that in master and remove the calls indeed, or create one in 15.0 to add a comment that this method is no more used in odoo codebase in 15.0 (and remove it in master forward port), as you prefer